console/src/components/WelcomePopup/WelcomePopup.js

```diff
@@ -0,0 +1,58 @@
+import Button from '@enact/agate/Button';
+import Divider from '@enact/agate/Divider';
+import kind from '@enact/core/kind';
+import FullscreenPopup from '@enact/agate/FullscreenPopup';
+import Group from '@enact/ui/Group';
+import {Panel, Panels} from '@enact/agate/Panels';
+import PropTypes from 'prop-types';
+import React from 'react';
+import SwitchItem from '@enact/agate/SwitchItem';
+
+import AppContextConnect from '../../App/AppContextConnect';
+
+const WelcomePopupBase = kind({
+	name: 'WelcomePopup',
+
+	propTypes: {
+		onClose: PropTypes.func,
+		updateUser: PropTypes.func,
+		userId: PropTypes.number
+	},
+
+	render: ({onClose, updateUser, userId, ...rest}) => (
+		<FullscreenPopup {...rest} onClose={onClose}>
+			<Panels>
```

Comment:
Is `<Panels>` necessary here for `<Panel>` to work or is this a placeholder for future panels?

Reply:
This is a placeholder. In a related feature that I'm working on, I'm adding additional panels which are the views for the welcome screen.